lua/lspconfig/configs/pkl_lsp.lua

```diff
@@ -0,0 +1,23 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'java', '-jar', 'path/to/pkl-lsp.jar' },
```

Comment:
The path depends on the user, the path here is just a placeholder.

Updated to match how similarly-configured LSPs are set up.

Reply:
what's the point of this config if it requires the user to manually configure the path?

Reply:
I looked at a couple other java-based LSPs for inspiration ([nextflow_ls](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/configs/nextflow_ls.lua), [groovyls](https://github.com/neovim/nvim-lspconfig/blob/master/lua/lspconfig/configs/groovyls.lua#L8) if you're curious).

I'm happy to remove `cmd` from `default_config`, especially since the doc covers it. Let me know what you think fits best :)